### ISSUE TITLE
Fully Kiosk plus trouble 

### ISSUE THREAD

**Issue by kisjeth** [Clooos/Bubble-Card#535]
![17167218656806400232639856585041](https://github.com/Clooos/Bubble-Card/assets/91030699/eb1dd2a5-d3a3-45be-b83e-293552535ed7)
![17167218851775482774795393722909](https://github.com/Clooos/Bubble-Card/assets/91030699/517ec3ba-a32f-441a-9ea6-246d05179dfe)
As you can see from the attached video when I press a button to invoke a pop-up it completely reloads the entire page. This problem does not occur when I use the companion app.This is the plus version of Fullykiosk 
![17167218460391127146997477282951](https://github.com/Clooos/Bubble-Card/assets/91030699/c7c5be00-1265-4174-b9c3-ee556f70e87c)

**Comment by Clooos** [comment]
Hi, can you tell me what Bubble Card version you have installed?

**Comment by kisjeth** [comment]
1.7.3

**Comment by Clooos** [comment]
Can you try the latest beta? This issue should be fixed on it and this is the candidate for the stable v2.0.0 release.

You can get it by going to HACS > Front end > Bubble Card. Then click the 3 dots on the top right then redownload. Toggle on "Show beta versions".

**Comment by kisjeth** [comment]
I tried beta11 but it doesn't work there.

**Comment by Clooos** [comment]
Are you sure that your cache was fully cleared?

**Comment by kisjeth** [comment]
Yes. I also force-closed the application and restarted it. Always the same.

**Comment by Clooos** [comment]
Just to be sure, have you checked in the editor if it shows the right version number?

**Comment by kisjeth** [comment]
![Screenshot_2024-05-27-19-49-35-394-edit_io homeassistant companion android](https://github.com/Clooos/Bubble-Card/assets/91030699/b5374fd5-fb85-4242-9246-e96bf76e1a72)

**Comment by Clooos** [comment]
Thank you for the confirmation, my guess is that this is some kind of incompatibility with Fully Kiosk, maybe some card you put inside some pop-ups?

I've tried my full Bubble Card dashboard on Fully Kiosk Browser and I have no issue at all.

**Comment by kisjeth** [comment]
As long as I used the horizontal stack there were no problems. Now I actually invoke pop-ups via buttons that activate vertical stacks via URL.Surely this is the problem. I don't know if you had foreseen this possibility in the code

**Comment by Clooos** [comment]
I understand now, this issue seems related to this one:

https://github.com/Clooos/Bubble-Card/issues/498

**Comment by kisjeth** [comment]
No, it's not referring to that error.I basically created many vertical stacks, which are invisible, and to invoke them I press a button which, instead of taking me back to the default action, connects to a URL (#xxxx)

**Comment by kisjeth** [comment]
I used the tile card. Later I'll try to change the card and use a basic one, and see how it behaves.

This is the card configuration:

```
type: tile
color: accent
show_entity_picture: false
vertical: true
hide_state: false
entity: sensor.camera_nostra_temperature
name: Camera nostra
tap_action:
  action: url
  url_path: '#camera_nostra'
icon_tap_action:
  action: url
  url_path: '#camera_nostra'
icon: mdi:bed-double
```

**Comment by Clooos** [comment]
The weirdest thing is that you said that it was working with the horizontal buttons stack. I guess that there is no reasons that the tile card is causing this.

**Comment by kisjeth** [comment]
I know, it's the same thing I tell myself. Because it doesn't make sense that it works one way and not another, when the process behind it is the same.

**Comment by kisjeth** [comment]
I installed an alternative version of fullykiosk on the mobile and was able to film the behavior based on the type of button triggered. As you can see in the case of the horizontal chaff it automatically opens the popup, however when I use a button it opens a new window.


https://github.com/Clooos/Bubble-Card/assets/91030699/e5998bf9-ff72-4bbe-b3dd-b89f2c2f760f

**Comment by kisjeth** [comment]
So, I installed an alternative kiosk version that allowed me to record the difference between the two triggers: when I press the horizontal stack button the popup opens normally, but when I try to recall it with the alternative button as you see above in the red circle, it opens a new page.


https://github.com/Clooos/Bubble-Card/assets/91030699/8e2a5db0-bcb0-4223-8dc6-0e2be7667c45

**Comment by Clooos** [comment]
I've found what is causing that, you need to use `action: navigate` and not `action: url`.

That should fixed your issue 🙂
